frontend/src/scenes/notebooks/Nodes/NotebookNodeExperiment.tsx

```diff
@@ -100,8 +103,13 @@ const Component = (props: NotebookNodeViewProps<NotebookNodeExperimentAttributes
                         size="small"
                         icon={<IconFlag />}
                         onClick={() => {
-                            insertAfter(buildFlagContent(experiment.feature_flag?.id || 'new'))
+                            if (nextNode?.type.name !== NotebookNodeType.FeatureFlag) {
```

Comment:
If you were interested we could look to expose this function:
https://github.com/PostHog/posthog/blob/35d440aac55b22c4a14f6866da4018fb65ab62e0/frontend/src/scenes/notebooks/Notebook/Editor.tsx#L255
That way you could decide to scroll the user to the existing feature flag rather than always inserting a new node.

Reply:
interesting, thanks!

Reply:
I think it's ok for now, but will come back to this, re-inserting is good for starting a new exploration without any moving nodes around schenanigans.